### PR TITLE
Update publicIpPrefix.json

### PR DESCRIPTION
**Pull request by khannarheams**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/publicIpPrefix.json [hunk 1]
```diff
@@ -359,6 +359,10 @@
           "format": "int32",
           "description": "The Length of the Public IP Prefix."
         },
+        "loadBalancerFrontendIpConfiguration": {
+          "$ref": "./network.json#/definitions/subResource",
```
Comment (nschonni) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/publicIpPrefix.json]:
```suggestion
          "$ref": "./network.json#/definitions/SubResource", 
```

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/publicIpPrefix.json [hunk 2]
```diff
@@ -359,6 +359,10 @@
           "format": "int32",
           "description": "The Length of the Public IP Prefix."
         },
+        "loadBalancerFrontendIpConfiguration": {
```
Comment (EvgenyAgafonchikov) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/publicIpPrefix.json]: Please update examples with this property.
Also, is it backreference or can we specify this value in payload? If the first one, should it be readOnly?

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/publicIpPrefix.json]: `loadBalancerFrontendIpConfiguration` is marked as read-only in backend model, so @khannarheams please add `readOnly: true`

**Comment by sergey-shandar** [comment]
@khannarheams are you still working on this PR?

**Comment by MikhailTryakhov** [comment]
need to be merged to mothly branch. closing due to inactivity

**Comment by number213** [comment]
@sergey-shandar @MikhailTryakhov added this change in this PR: https://github.com/Azure/azure-rest-api-specs/pull/6693

**Pull request closed without merging**